### DISCUSSION TITLE
People Search improvement

### DISCUSSION THREAD

**Discussion by AnttiHal** [community/community#101557] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hi!
This is more related to oragnizational level but there wasn't accurate topic for that so here it goes:
On organization level, people tab:
Now when you search people, the results are based only to username (or the real name if user has decided to show that). Could this be expanded also to emails (if they exist)? It would help A LOT! 
Thank you in advance!
Antti

**Comment by Shahnawazk7944** [comment]
hey @AnttiHal 
Regrettably, at present, GitHub organizations lack the built-in capability to search for individuals by email address. The current search function solely takes into account usernames and real names (if publicly visible) as far as I know.
